lib/msal-browser/src/cache/BrowserStorage.ts

```diff
@@ -23,7 +23,7 @@ export class BrowserStorage extends CacheManager {
     // Window storage object (either local or sessionStorage)
     private windowStorage: Storage;
     // Client id of application. Used in cache keys to partition cache correctly in the case of multiple instances of MSAL.
-    private clientId: string;
+    public clientId: string;
```

Comment:
I don't really like this change, but I needed a way to obtain the `clientId` in the interaction handler class after a redirect and before calling into the `acquireToken` APIs. Any thoughts?

Reply:
This usage can be seen in the other changes to msal-browser in this PR

Reply:
This technically makes it a public setter. I would keep it private then add a public getter function (e.g. getClientId)

Reply:
You should be able to get it from the cached request in the `RedirectHandler`

Reply:
I created a `getClientId()` for now. @tnorling It looks like the cached request is an `AuthorizationCodeRequest` which doesn't contain `clientId`, is there something else?

Reply:
Ah you're right. Thought it was part of `BaseAuthRequest`. You could also pass it in from `PublicClientApplication` since it will be part of the config, but not sure what's cleaner

Reply:
Going to leave the `getClientId()` for now.